### PR TITLE
Migrated OpenCV to 3.1 (default in Kinetic)

### PR DESCRIPTION
**Pull request by bdholt1**
I made the changes required to compile and run mapviz on kinetic.  This is one of them.

### CODE REVIEW

**Comment by evenator** [comment]
Relevant:

* [Kinetic Migration Guide: OpenCV](http://wiki.ros.org/kinetic/Migration#OpenCV)
* [ROS Wiki: OpenCV3](http://wiki.ros.org/opencv3)

File: swri_transform_util/package.xml [hunk 1]
```diff
@@ -13,13 +13,12 @@
...
   
   <buildtool_depend>catkin</buildtool_depend>
-  
+
   <depend>boost</depend>
   <depend>diagnostic_msgs</depend>
   <depend>geographic_msgs</depend>
   <depend>geometry_msgs</depend>
   <depend>libgeos++-dev</depend>
-  <depend>libopencv-dev</depend>
```
Comment (pjreed) [inline swri_transform_util/package.xml]: If we're not using the system-provided OpenCV 2 anymore, it makes sense to remove this, but would it make sense to add a rosdep key for the ROS-provided OpenCV 3?

Comment (bdholt1) [inline swri_transform_util/package.xml]: Based on http://wiki.ros.org/opencv3 is seems that the key is opencv3, so I'll add that back.

Comment (evenator) [inline swri_transform_util/package.xml]: See http://wiki.ros.org/opencv3#package.xml

Comment (bdholt1) [inline swri_transform_util/package.xml]: Sorry, I was too quick on the trigger, I should have read the whole document first.

**Comment by bdholt1** [comment]
Is there anything else that need to be done before this can be merged?

**Comment by pjreed** [comment]
This looks good to me, although I'd like to get somebody else to test it out first (I don't have a Kinetic environment right at the moment), and it couldn't hurt to get @malban's approval.

**Comment by malban** [comment]
Looks good to me.  We haven't tested it since we don't use kinetic yet, but I'll go ahead and merge it anyway.

**Pull request merged**